src/shared/components/query/QueryAndDownloadTabs.tsx

```diff
@@ -139,7 +139,7 @@ export default class QueryAndDownloadTabs extends React.Component<
                         }}
                     >
                         <div>
-                            <QuickSearch />
+                            <QuickSearch queryStore={this.store} />
```

Comment:
rather than making querystore a depedency of QuickSearch, lets just pass the study list in as a prop.    So you could test against isCompete out here and then pass in the resulting list.